homeassistant/components/triggercmd/hub.py

```diff
@@ -0,0 +1,65 @@
+"""A 'hub' that connects several devices."""
+
+from __future__ import annotations
+
+import asyncio
+import logging
+
+import jwt
+from triggercmd import client
+
+from homeassistant.core import HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Hub:
```

Comment:
The hub class is not depending on any HA objects and can therefore be moved to the lib

Reply:
I moved it to the library.